core/bubbles/textinput_bubble.ts

```diff
@@ -121,7 +121,7 @@ export class TextInputBubble extends Bubble {
 
     const body = document.createElementNS(dom.HTML_NS, 'body');
     body.setAttribute('xmlns', dom.HTML_NS);
-    body.className = 'blocklyMinimalBody';
+    body.className = 'blocklyMinimalBody blocklyBubbleComment';
```

Comment:
Text input bubbles are no longer comment specific, so I would prefer to avoid using `Comment` in the CSS class.

Instead of adding a new class you use the existing `blocklyTextArea` or `blocklyTextInputBubble`? And by "use" I mean delete this and change the CSS to reference them, because they're already added to some elements.

If not, can you change this to reference being a text input bubbles instead of a comment?

Reply:
Changed.  I'd initially been extremely nervous to do something like this because the CSS path `.blocklyTextInputBubble textarea` crosses a 'foreignobject' boundary.  All the browsers I've tested this on seem ok with this now.  But historically that would have been asking for trouble.

Reply:
Oh good to know for future reference! I'll start checking that.